showcase/src/app/shared/components/example-viewer/example-viewer.component.ts

```diff
@@ -22,13 +28,23 @@ export class ExampleViewerComponent implements OnInit {
 	public filesPath = "undefined";
 	@Input()
 	public exampleTitle = "undefined";
+	@Input()
+	public id = "";
+
+	public get exampleState(): string {
+		return this.routingService.getCurrentState().name;
```

Comment:
I know that defining some properties as getters is more concise sometimes, however in this case I think we should use a simple property to keep it consistent with the rest of properties in this class. What do you think?

Reply:
The only problem that could happen is when the state changes during the lifetime of the component. But I suppose that is unlikely

Reply:
Yeah, very very unlikely IMO ;)